config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
+
+# Specify a serializer for the signed and encrypted cookie jars.
+# Valid options are :json, :marshal, and :hybrid.
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
Rails 3 used "marshal" encoding for cookies on the users' computers. We missed upgrading to json in rails 4, but can upgrade them now.

However, if we end up reverting this branch, we will have to make sure we leave the serializer as :json or :hybrid, or users will get server errors when they send marshal-encoded cookies

Once a user with a cookie visits our server one time, the hybrid encoder will change their cookie to be JSON instead, so at some point we can move to being all json all the time.